woocommerce/woocommerce
# Create ProductForm component to merge duplicated UI

### All Submissions:






### Changes proposed in this Pull Request:



Creates a new component ProductForm to merge duplicated UI that existed between AddProductPage and EditProductPage.

No behavior should change.

Closes #35766.






### How to test the changes in this Pull Request:



1. Create a new product in New Product Management Experience.
2. Fill out all fields.
3. Click on 'Publish'.
4. You will be redirected to the Edit Product page.
5. Change all/some of the fields data.
6. Click 'Update'
7. Check if the Update occurred successfully




### Other information:




### FOR PR REVIEWER ONLY:
